src/ensembl/src/content/app/custom-download/containers/content/attributes-accordion/state/attributesAccordionActions.ts

```diff
@@ -15,7 +26,7 @@ export const setAttributes = createAsyncAction(
 )<undefined, {}, Error>();
 
 export const fetchAttributes: ActionCreator<
-  ThunkAction<void, any, null, Action<string>>
+  ThunkAction<void, Attributes, null, Action<string>>
```

Comment:
Second type of ThunkAction is root state (that is returned by the getState function). It can’t be `Attributes`

Reply:
Updated it.